Scripts/childControl.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class childControl : MonoBehaviour
```

Comment:
```suggestion
public class ChildControl : MonoBehaviour
```

Class isimleri Pascal Case olur.